{"uuid": "82b18131-7873-44fd-b063-d70c9a8e89bd", "children": ["bbbdc1d4-32ac-479b-83e1-70f56e925e7f", "2d625903-b127-4c6e-b25f-02a9fadbd4a5"], "befores": [{"name": "init_web_driver", "status": "passed", "start": 1689703874767, "stop": 1689703877830}], "afters": [{"name": "init_web_driver::0", "status": "broken", "statusDetails": {"message": "applitools.common.errors.DiffsFoundError: Test 'Check visually the users chart' of 'Grafana - users chart' detected differences! See details at: https://eyes.applitools.com/app/batches/00000251712596903506/00000251712596903293?accountId=uC7pT3e6R0qz0R2dpXaRFA~~\n", "trace": "  File \"C:\\Python311\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 917, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\BarKadosh-Cello\\PycharmProjects\\test_automation_final_project\\test_cases\\conftest.py\", line 66, in init_web_driver\n    eyes.close()  # Applitools\n    ^^^^^^^^^^^^\n  File \"C:\\Python311\\Lib\\site-packages\\applitools\\selenium\\eyes.py\", line 197, in close\n    return self._close(raise_ex, True)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python311\\Lib\\site-packages\\applitools\\selenium\\eyes.py\", line 533, in _close\n    results = self._commands.eyes_get_results(self._eyes_ref, raise_ex)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python311\\Lib\\site-packages\\applitools\\selenium\\command_executor.py\", line 254, in eyes_get_results\n    return self._checked_command(context, \"Eyes.getResults\", payload)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python311\\Lib\\site-packages\\applitools\\selenium\\command_executor.py\", line 267, in _checked_command\n    _check_error(response_payload)\n  File \"C:\\Python311\\Lib\\site-packages\\applitools\\selenium\\command_executor.py\", line 277, in _check_error\n    raise usdk_error\n"}, "start": 1689703909667, "stop": 1689703915904}], "start": 1689703874767, "stop": 1689703915904}